src/lib/node/net.js

```diff
@@ -685,9 +685,9 @@ foam.CLASS({
       if ( this.port ) options.port = this.port;
```

Comment:
Ignore file: Feature required universal HTTPRequest. Will merge in #248 after it's reviewed.